src/components/checkbox/checkbox.tsx

```diff
@@ -94,6 +96,40 @@ export class Checkbox {
     @Prop({ reflect: true })
     public required: boolean = false;
 
+    /**
+     * The label to show, when the component is `readonly` and its `value` is `true`.
+     * This can be used to clarify what kind of data is being visualized.
+     * If not set, the `label` property will be used.
+     */
+    @Prop({ reflect: true })
+    public readonlyTrueLabel?: string;
+
+    /**
+     * The label to show, when the component is `readonly` and its `value` is `false`.
+     * This can be used to clarify what kind of data is being visualized.
+     * If not set, the `label` property will be used.
+     */
+    @Prop({ reflect: true })
+    public readonlyFalseLabel?: string;
+
+    /**
+     * The icon to show, when the component is `readonly` and its `value` is `true`.
+     * This can be used to clarify what kind of data is being visualized.
+     * If not set, a default icon with a default color will be used.
+     * Colors can be customized using the `Icon` interface.
+     */
+    @Prop({ reflect: true })
+    public readonlyTrueIcon?: string | Icon;
+
+    /**
+     * The icon to show, when the component is `readonly` and its `value` is `false`.
+     * This can be used to clarify what kind of data is being visualized.
+     * If not set, a default icon with a default color will be used.
+     * Colors can be customized using the `Icon` interface.
+     */
+    @Prop({ reflect: true })
+    public readonlyFalseIcon?: string | Icon;
+
```

Comment:
A config like this:
```json
{
    "readonlyTrueLabel": "Still works here",
    "readonlyFalseLabel": "Has quit their job",
    "readonlyTrueIcon": {
        "name": "inactive_state",
        "color": "rgb(var(--color-gray-default))"
    },
    "readonlyFalseIcon": {
        "name": "in_progress",
        "color": "rgb(var(--color-sky-default))"
    }
}
```
Doesn't do anything in Lime Admin at all.

I think these new props should somehow be added to the `extraProps` of the checkbox widget 👇 or somewhere in this file

https://github.com/Lundalogik/lime-elements/blob/0f56884b79f3149732d2710cd5ee327cb5a9b9b1/src/components/form/widgets/checkbox.ts#L20

### Other relevant and similar issues:
1. It's also worth nothing that adding `help` which we did here: https://github.com/Lundalogik/lime-elements/pull/2622, is not possible to do via Lime Admin on any component, but writing it in the schema in the code works:
    <img width="40%" src="https://github.com/Lundalogik/lime-elements/assets/35954987/1a6bc205-18c5-40ac-a109-4afa5a08dc58)"/> <img width="40%" src="https://github.com/Lundalogik/lime-elements/assets/35954987/50e011d2-00df-4623-b1ff-0168c36d7b85"/>

2. Adding `helperText` on the boolean field is not possible via Lime Admin.
3. Adding styles (such as CSS variables of components) is not possible on any field via Lime Admin.

I think same things are wrong with out implementations about these.

Maybe you guys can help @adrianschmidt @vicgeralds @jgroth

Reply:
There's an issue about the CSS variables here Lundalogik/crm-feature#3772
I thought it was possible to specify props for the components in lime-admin, but maybe we need to fix some bug there as well :thinking:

Reply:
It is possible to specify props for the component, and it also works (at least for text fields) without specifying a component name.

The problem with `help` is that it's not a property to pass. It's only a feature of limel-form when (ab)using schemas. :wink:

> Adding helperText on the boolean field is not possible via Lime Admin.

This should work, but I can see that it doesn't when trying it out. Not sure yet why 😕

Reply:
> The problem with help is that it's not a property to pass. It's only a feature of limel-form when (ab)using schemas. 😉

😮 Then we completely missed the target @civing @john-traas . One of the most important reasons for adding this was to be able to add contextual help to form elements, both on cards (by consultants) and in admin (link to platform docs by us).

Reply:
The reason specifying props for a boolean field didn't work, is that we don't pass a default custom component name, which we do apparently for text fields, for example. So it works if you specify limel-checkbox as component name.

Reply:
> The reason specifying props for a boolean field didn't work, is that we don't pass a default custom component name, which we do apparently for text fields, for example. So it works if you specify limel-checkbox as component name.

The problem with that is that `limel-checkbox` does not have a `value` prop (it has `checked` instead :disappointed: )

Reply:
The reason limel-checkbox has `checked` instead of `value` is that that's what a native html checkbox has, and my idea originally was to keep Lime Elements' components as close to drop-in replacements as possible.

It feels like that was less possible than I originally intended, and I think we have already more or less decided to move away from that idea.

Adding a `value` property that takes precedence over `checked` ought to be simple, and I would suggest that we deprecate `checked` at the same time, if you all agree.

Ping @Lundalogik/lime-elements-maintainers.

Reply:
> 😮 Then we completely missed the target @civing @john-traas . One of the most important reasons for adding this was to be able to add contextual help to form elements, both on cards (by consultants) and in admin (link to platform docs by us).

Right now this is a feature of lime-forms where you specify the inputs for the help component in the schema, yes.

If we would like to make this generic for all components, we need to either support it as props for every component or do some other clever shenanigans. I really dislike the idea of violating DRY by extending every component with the same props.

Anyhow, we have the helper component so we can in theory add it somehow to components outside of lime-form.